integreat_cms/cms/templates/events/event_list.html

```diff
@@ -69,7 +69,7 @@ <h1 class="heading">{% trans 'Events' %}</h1>
             {% if events %}
                 {% for event in events %}
                     {% get_translation event language.slug as event_translation %}
-                    {% include "events/event_list_row.html" with event=event event_translation=event_translation %}
+                    {% include "events/event_list_row.html" with event=event event_translation=event_translation %}              
```

Comment:
```suggestion
                    {% include "events/event_list_row.html" with event=event event_translation=event_translation %}
```
I see no reason to add a bunch of spaces here.